packages/paste-website/src/components/theme-switcher/index.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+interface ThemeSwitcherProps {
+  children?: React.ReactElement;
+}
+export const ThemeSwitcher: React.FC<ThemeSwitcherProps> = () => {
+  const [theme, setTheme] = React.useState('console');
```

Comment:
We should use a constant for `'console' 'sendgrid'` here.

Also console will eventually map to default, right?

Reply:
I'm throwing out the theme switcher stuff for this release